### DISCUSSION TITLE
Release build download?

### DISCUSSION THREAD

**Discussion by spyhunter99** [apache/camel-karavan#819] [General]
I'm having some issues compiling karavan. Is there a release build available to download? i'm just looking for a standalone setup for test/eval, not a cloud/container builder

**Comment by mgubaidullin** [comment]
Karavan cloud application requires to he run inside kubernetes. There is no standalone version.
However we are working on this feature for the next release.

**Comment by davsclaus** [comment]
there is a docker based karavan you can use for trying karavan
